src/cacheServerClient/cacheServerClient.ts

```diff
@@ -96,20 +101,24 @@ export class CacheServerClient implements ICacheServerClient {
     return data.orgs;
   }
 
-  async getOrganizationsBySearchPhrase({ search }: { search: string }) {
-    const { data } = await this.httpClient.get<IOrganization[]>(
-      `/namespace/search/${search}?type=Org`
-    );
-    return data;
-  }
-
-  async getApplicationsBySearchPhrase({ search }: { search: string }) {
-    const { data } = await this.httpClient.get<IApp[]>(`/namespace/search/${search}?type=App`);
-    return data;
-  }
-
-  async getRolesBySearchPhrase({ search }: { search: string }) {
-    const { data } = await this.httpClient.get<IRole[]>(`/namespace/search/${search}?type=Role`);
+  async getNamespaceBySearchPhrase({
+    types,
+    search
+  }: {
+    types?: ENSNamespaceTypes[];
+    search: string;
+  }) {
+    if (types && types.length > 0) {
+      const searchTypes = types.map(type => type.charAt(0).toUpperCase() + type.slice(1));
+      const { data } = await this.httpClient.get<IOrganization[]>(`/namespace/search/${search}`, {
```

Comment:
I'm a little confused. Maybe it doesn't matter as data is being operated on but this `httpClient` call always returns `IOrganization[]` even though the function itself could return `IOrganization[] | IApp[] | IRole[]` based on `ICacheServerClient`. Should the type in the angle brackets be `IOrganization[] | IApp[] | IRole[]`?

Reply:
Yeah, sorry my mistake I forgot about this :)

Reply:
Fixed :)